src/Symfony/Component/Filesystem/Tests/PathTest.php

```diff
@@ -439,25 +439,6 @@ public function testGetRoot(string $path, string $root)
         $this->assertSame($root, Path::getRoot($path));
     }
 
-    public function providePathTests(): \Generator
```

Comment:
why removing this one?

Reply:
Not sure, investigating

Reply:
What I can see is, that the next method
```php
private static function getPathTests(): \Generator
```

contains the exact same entries, but uses `yield from` instead of `yield` per entry. Maybe a merge issue?

Reply:
I checked it locally and the dataProvider is not used anymore. 
I wasn't aware, that rector removes them.... 

I created a PR
* https://github.com/symfony/symfony/pull/49278